packages/template-ui/src/views/Home.vue

```diff
@@ -17,7 +17,16 @@
       </b-container>
       <FilterContent v-if="hasFilters" />
 
-      <div v-if="isFilterEmpty">
+
+      <div v-if="hasFlatGrid">
+        <CardGrid
+          variant="collapsible"
+          :nodes="contentNodes"
+          :mediaQuality="mediaQuality"
+          :cardColumns="cardColumns"
+        />
+      </div>
```

Comment:
Nitpick: could use an extra newline to line up with the `<div v-else>` later on :)

```suggestion

      </div>
```